input/includes/servicerequest_notes.md

```diff
@@ -0,0 +1,27 @@
+#### Mandatory Search Parameters:
+
+The following search parameters and search parameter combinations **SHALL** be supported:
+
+1. **SHALL** searching using the **[`_id`](https://hl7.org/fhir/R4/servicerequest.html#search)** search parameter:
+ 
+    `GET [base]/ServiceRequest?_id=[id]`
+
+    Example:
+    
+      1. GET [base]/ServiceRequest?_id=2169591
+
+    *Implementation Notes:* Fetches a bundle with the requested ServiceRequest, instead of just the resource itself, and allows for the inclusion of additional search parameters such as _include, _revinclude, or _lastUpdated ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id))
+
+
+1. **SHALL** support searching using the **[`requisition`](https://hl7.org/fhir/R4/observation.html#search)** search parameter:
```

Comment:
Incorrect link, should link to https://hl7.org/fhir/R4/servicerequest.html#search

Reply:
fixed